doc/classes/Color.xml

```diff
@@ -370,7 +371,7 @@
 		<method name="to_argb32" qualifiers="const">
 			<return type="int" />
 			<description>
-				Returns the color converted to a 32-bit integer in ARGB format (each byte represents a color channel). ARGB is more compatible with DirectX.
+				Returns the color converted to a 32-bit integer in ARGB format (each component is 8 bits). ARGB is more compatible with DirectX.
```

Comment:
> ARGB is more compatible with DirectX.

That's it? Could there be more to say? Otherwise this method (and related) look to exist for no discernible reason.